packages/components/src/Combobox/components/ComboboxAction/ComboboxAction.tsx

```diff
@@ -1,24 +1,39 @@
-import React from "react";
+import React, { useContext } from "react";
 import styles from "./ComboboxAction.css";
 import { Typography } from "../../../Typography";
 import { ComboboxActionProps } from "../../Combobox.types";
+import { ComboboxContext } from "../../ComboboxProvider";
+
+export function ComboboxAction(props: ComboboxActionProps) {
+  const { searchValue } = useContext(ComboboxContext);
+
+  if (props.visible) {
+    const isVisible =
+      typeof props.visible === "function" && !props.visible({ searchValue });
+
+    if (isVisible || !props.visible) {
+      return null;
+    }
+  }
+
+  const options = { searchValue };
+  const computedLabel =
+    typeof props.label === "string" ? props.label : props.label(options);
 
-export function ComboboxAction(props: ComboboxActionProps): JSX.Element {
   return (
     <div className={styles.actionContainer}>
       <button
         className={styles.actionButton}
-        onClick={props.onClick}
+        onClick={e => props.onClick(e, options)}
         type="button"
-        aria-label={props.label}
```

Comment:
No more `aria-label`?

What used to show up - under ARIA Attributes in the inspector:
<img width="942" alt="Screenshot 2023-12-04 at 5 03 21 PM" src="https://github.com/GetJobber/atlantis/assets/104797704/fcc2e899-cb07-4adc-9e42-7cba8a0c79f4">

What we have now:
<img width="457" alt="Screenshot 2023-12-04 at 5 03 43 PM" src="https://github.com/GetJobber/atlantis/assets/104797704/821ba075-b66c-4334-a134-17d0f931f1a3">

Reply:
Aria's are needed for when it's different from the label (or lack of it). it looks like they're always the same so aria's aren't really needed. Forgot to add a comment on it haha